doc/source/gitwash/development_workflow.rst

```diff
@@ -334,7 +334,7 @@ Rewriting commit history
 
    Do this only for your own feature branches.
 
-There's an embarassing typo in a commit you made? Or perhaps the you
+There's an embarrassing typo in a commit you made? Or perhaps the you
```

Comment:
I just can't remember whether this was a joke or not.  But - oh well.

Reply:
Ah, didn't notice the context! Perhaps I can leave it then.

Reply:
If you can be bothered, I enjoy the irony...

Reply:
Done